### ISSUE TITLE
Confirming confirm page too fast results in null URL

### ISSUE THREAD

**Issue by jomo** [mozilla/multi-account-containers#1027]
Steps to reproduce:

0. Configure a site to 'Always open in $container'
0. Open new tab with that site
0. Press enter while or immediately after confirm-page.html opens

confirm-page.html is now shown again, but with `null` URL and null Container:

![screenshot](https://user-images.githubusercontent.com/3585743/33577335-eeedd8ae-d942-11e7-8c07-53453d4d1328.png)

I didn't look at the code, but one possible cause might be that pressing the button too fast (before the actual URL is filled in?) redirects to empty string (or the like), thus redirecting to the confirm-page with no URL parameters.

**Comment by sakshi97** [comment]
Hi @groovecoder. I would like to take up this issue if it's free.

**Comment by bini11** [comment]
Hey @groovecoder I am an outreachy applicant and would like to work on this bug if it's free.

**Comment by oksmelnik** [comment]
I'd like to work on this issue (outreachy applicant)

**Comment by oksmelnik** [comment]
actually, can't reproduce it. I'm redirected from the confirm page to the page url when I press enter. Do you still have this bug, @jomo?

**Comment by jomo** [comment]
Yes, it's still happening when you press enter fast enough. Using Version 6.1.1.

I typed the URL into the URL bar and kept pressing Enter really fast. I was able to reproduce it a couple times.

**Comment by oksmelnik** [comment]
Thanks! See it now as well

**Comment by grahamperrin** [comment]
> `null` URL 

Maybe fixed in a recent release of Firefox. Please review.
